preferences/preference.json

```diff
@@ -51,7 +51,7 @@
       "text": "Generate Foregin Key Constraints",
       "description": "Generate Foreign Key constraint when reference field is populated.",
       "type": "check",
-      "default": false
+      "default": true
```

Comment:
This seems like the correct default behavior.

Reply:
Correct